model/buildlogger_test.go

```diff
@@ -593,10 +593,21 @@ func TestBuildloggerFindLogs(t *testing.T) {
 		assert.NoError(t, db.Collection(buildloggerCollection).Drop(ctx))
 	}()
 	log1, log2 := getTestLogs()
-
-	_, err := db.Collection(buildloggerCollection).InsertOne(ctx, log1)
-	require.NoError(t, err)
-	_, err = db.Collection(buildloggerCollection).InsertOne(ctx, log2)
+	time.Sleep(time.Millisecond)
```

Comment:
Rather than relying on sleeping to change `time.Now()`, would it be better to just pass in a time to `getTestLogs()`?